src/esbmc/options.cpp

```diff
@@ -387,7 +387,10 @@ const struct group_opt_templ all_cmd_options[] = {
      "results in a verification failure when its call is reachable"},
     {"conv-assert-to-assume",
      NULL,
-     "convert assertions for bounds and pointer checks into assumptions"}}},
+     "convert assertions for bounds and pointer checks into assumptions"},
+    {"unknown-method-args-check",
+     NULL,
+     "check every argument to an unknown function call"}}},
```

Comment:
Without looking at the source code, it's not clear (to me) what this function does.